src/components/uielements/CasePreview.astro

```diff
@@ -1,133 +1,111 @@
-if (!logoimages[caseData.logo])
-  throw new Error(
-    `"${caseData.logo}" does not exist in glob: "src/assets/logos/{,**/}*.{jpeg,jpg,png,gif}"`,
-  );
 ---
 
-<div
-  class:list={[
-    "group flex flex-col justify-stretch gap-8 motion-safe:animate-fade-up motion-safe:animate-delay-1000",
-    small ? "items-stretch" : "w-full md:flex-row",
-  ]}
->
-  <div
-    class:list={[
-      "flex w-full items-center justify-center overflow-hidden rounded-t-xl transition-all motion-safe:group-hover:shadow-xl",
-      small ? "max-h-[400px]" : "md:w-1/2 md:rounded-l-xl md:rounded-tr-none",
-    ]}
-  >
-    <Picture
-      src={caseImage}
-      alt={caseData.altTextImage || ""}
-      widths={[400, 600, 790]}
-      sizes="(max-width: 790px) 100vw, 790px"
-      class="h-[unset] min-h-full w-auto min-w-full max-w-[unset] shrink-0 transition-transform duration-300 ease-in-out motion-safe:group-hover:scale-105"
-      loading="lazy"
-    />
-  </div>
-  <div
-    class:list={[
-      "flex max-h-full w-full flex-col justify-between gap-8 border-b-2 border-y-quaternary pb-8",
-      small ? "h-3/4 md:px-4" : "px-4 md:w-1/2 md:border-y-2 md:p-8",
-    ]}
-  >
+<div class:list={["motion-safe:animate-fade-up motion-safe:animate-delay-1000 group flex flex-col justify-stretch gap-8",
+  small ? "items-stretch" : "w-full md:flex-row"
+]}>
+  {
+    caseData.introImage &&
+    <div
+        class:list={["w-full flex items-center justify-center rounded-t-xl transition-all overflow-hidden motion-safe:group-hover:shadow-xl ",
+          small ? "max-h-[400px]" : "md:w-1/2 md:rounded-tr-none md:rounded-l-xl"
+        ]}>
+      <SanityImg
+          src={caseData?.introImage}
+          alt={caseData?.introImage?.asset?.altTexts?.[language] || ""}
+          widths={[400, 600, 790]}
+          sizes="(max-width: 790px) 100vw, 790px"
+          class:list={["motion-safe:group-hover:scale-105 transition-transform ease-in-out duration-300 min-w-full min-h-full max-w-[unset] h-[unset] w-auto",
+            small ? "object-scale-down shrink-0" : ""]}
+          loading="lazy"
+      />
+    </div>
+  }
+  <div class:list={["pb-8 max-h-full w-full flex flex-col justify-between gap-8 border-b-2 border-y-quaternary",
+    small ? "h-3/4 md:px-4" : "px-4 md:w-1/2 md:p-8 md:border-y-2"
+  ]}>
     <div class="flex flex-col gap-2">
       <div class="mb-4 flex flex-row items-start justify-between gap-8">
         <div class="flex flex-col">
-          <span class="font-serif text-primary" set:html={caseData.client} />
+          <span class="font-serif text-primary" set:html={caseData?.metadata?.title} />
           <h2
             class="caseheadline mb-2 font-serif text-xl [&>mark]:bg-none [&>mark]:bg-clip-border [&>mark]:text-black"
-            set:html={caseData.title}
+            set:html={caseData?.intro?.title}
           />
         </div>
         {
-          caseData.logo && (
-            <Image
+          caseData?.metadata?.image && (
+            <SanityImg
```

Comment:
if you're creating a SanityImage component, then this component can also fetch the alt tags always no? I think the alt should be resolved inside the component, together with the src.

Reply:
You should just pass i18nimage, classes, width/height, and loading to the SanityImage component and it should handle the rest.

Reply:
Doesn't seem to be the case, without the alt tag I don't see an alt tag. This is what you mean right?
```
            <SanityImg
              loading="lazy"
              src={caseData?.metadata?.image}
              class="h-12 w-auto shrink-0 sm:h-16 md:h-12 xl:h-16"
              height={60}
            />
```